packages/terra-dialog/src/DialogBody.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Dialog.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The children to be placed within the main content area of the dialog.
+   */
+  children: PropTypes.node.isRequired,
+};
+
+const defaultProps = {
+  children: null,
+};
+
+const DialogBody = ({ children }) => {
+  const DialogBodyClassNames = cx([
```

Comment:
This variable should probably be camelCase

Reply:
https://github.com/cerner/terra-core/pull/1209/commits/2fabef8d09a87699422ef4b44f00ce87220ae782